mbed_greentea/mbed_test_api.py

```diff
@@ -138,6 +139,8 @@ def run_host_test(image_path,
     @param max_failed_properties After how many unknown properties we will assume test is not ported
     @param enum_host_tests_path Directory where locally defined host tests may reside
     @param num_sync_packtes sync packets to send for host <---> device communication
+    @param tag-filters Filter list of available devices under test to only run on devices with the provided list
```

Comment:
This `@param` does not describe a parameter. Perhaps you meant `tags` instead of `tag-filters`?